applications/LinearSolversApplication/custom_python/add_custom_solvers_to_python.h

```diff
@@ -17,14 +17,6 @@
 
 // Project includes
 
-//MKL implements some of FEAST's function. This define excludes the definitions of MKL's FEAST
-//functions in mkl/include/mkl_solvers_ee.h. Otherwise, some functions are defined multiple times
-#if defined USE_EIGEN_FEAST
-// defining both as MKL uses either, depending on the version
-#define _MKL_SOLVERS_EE_H
-#define __MKL_SOLVERS_EE_H
-#endif
```

Comment:
@rubenzorrilla this was the issue, now I moved it to CMake so it should be working also when having the py interface split up over multiple files

FYI @qaumann

Reply:
Thanks a lot :heart: